### PR TITLE
Added test doc for sprint2 

### LINKED ISSUE #54: Add a test txt file
Refer to [sprint instructions](https://github.com/orgs/NCATComp410/teams/comp410summer2022class/discussions/12)

### PR DESCRIPTION
**Pull request by AntrayS**
closes #54

### CODE REVIEW

**Comment by claesmk** [comment]
Please also add a second txt file that doe not contain PII as discussed in the sprint instructions

> Testing the personally identifiable information (PII) scanner will require multiple sample files. For sprint 2, you should plan to create two new sample files of the type specified in your issue. This will be docx, pdf, xlsx, or txt (text). One file will contain some PII, the other file will not. This will better allow us to test the scanner to make sure it only finds files that contain PII and ignores others.

Please also add a unit test case that covers reading your txt files, similar to [this](https://github.com/NCATComp410/comp410_summer_2022/blob/787662907bfe00958b66fc425eb1516dbbedf939/test_scan.py#L96)

**Comment by claesmk** [comment]
Your tests are failing - please check the output and look into your indent levels

**Comment by AntrayS** [comment]
> Your tests are failing - please check the output and look into your indent levels

how do i know when the indentation is correct i keep getting the same thing

**Comment by AntrayS** [comment]
i cant figure out the indent error

File: scan.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 # Returns all the text in a file as a list
 # Return an empty list if no text is found
 def get_file_text(file) -> list:
+    """
```
Comment (claesmk) [inline scan.py:13]: It's OK if you want to do this, however, please make sure that rtype matches the type hint (-> list)

**Review by claesmk — CHANGES_REQUESTED** [review]

File: scan.py (hunk 1 above)
Comment (AntrayS) [reply scan.py:13]: thank you !

**Review by AntrayS — COMMENTED** [review]
change return type to a list

**Review by claesmk — APPROVED** [review]
Looks good

**Pull request merged**